.planning/milestones/v1.1-phases/10-accessibility-html-aria/10-01-PLAN.md

```
@@ -0,0 +1,285 @@
+     - If from WordPress nonce: pass a unique name to `wp_nonce_field()`
+     - If from multiple hook registrations: ensure the hook runs only once
+
+  3. Check `footer.php` line 30 -- the newsletter form has `wp_nonce_field( 'skyyrose_newsletter', 'skyyrose_newsletter_nonce' )`. If the same nonce field name appears elsewhere (header, template), it would create duplicate IDs. Search for all `skyyrose_newsletter_nonce` usages and ensure uniqueness.
+
+  Note: This is an audit-and-fix task. If no duplicate IDs are found, document that finding in the verification output. The rendered HTML depends on which templates load per page, so focus on IDs that could collide within a single page view.
+  </action>
+  <verify>
+    <automated>cd wordpress-theme/skyyrose-flagship && python3 -c "
+import re, os, sys
+
+# Collect all hardcoded id= values from PHP files in template-parts that are included in loops
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)